react/features/app/actions.web.ts

```diff
@@ -129,6 +129,14 @@ export function appNavigate(uri?: string) {
 
         dispatch(setLocationURL(locationURL));
         dispatch(setConfig(config));
+
+        if (Number(API_ID) === API_ID && getState()['features/base/config'].disableIframeAPI) {
```

Comment:
Why the check for API_ID? We have some isInIframe helpers, wouldn't that be more explicit?

Reply:
Cause the same check exists in the same file. I was not aware of helpers. I will check it out.

Reply:
Done.